Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1531,10 +1531,15 @@ def prepare_HLT(self, sequence = None):
 		else:
 			self.executeAndRemember('process.loadHltConfiguration("%s",%s)'%(sequence.replace(',',':'),optionsForHLTConfig))
         else:
```

Comment:
this is a temporary workaround - the final solution is to properly handle the HLT schedule (heare and/or in the code that creates it)